utils/frame/benchmarking-cli/src/machine/hardware.rs

```diff
@@ -18,8 +18,64 @@
 //! Contains types to define hardware requirements.
 
 use lazy_static::lazy_static;
-use serde::{Deserialize, Serialize};
-use std::fmt;
+use sc_sysinfo::{Throughput, Unit};
+use serde::{
+	de::{Error, MapAccess, Visitor},
+	ser::SerializeMap,
+	Deserialize, Deserializer, Serialize, Serializer,
+};
+use sp_std::{fmt, fmt::Formatter};
+
+pub fn serialize_throughput<S>(t: &Throughput, serializer: S) -> Result<S::Ok, S::Error>
+where
+	S: Serializer,
+{
+	let mut map = serializer.serialize_map(Some(1))?;
+	let (value, unit) = t.normalize();
+	let unit_as_str = match unit {
```

Comment:
```suggestion
	let unit_as_str = format!("{}", unit);
```
An `Unit` should implement `Display` IIUC.

Reply:
Talked to @koute its fine.